File: 1703_2/app/src/main/java/com/example/a1703_2/MainActivity.java
package com.example.a1703_2;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button okButton = findViewById(R.id.ok);
        okButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                EditText text = findViewById(R.id.entry);
                String value = text.getText().toString();
                Toast.makeText(getApplicationContext(), value, Toast.LENGTH_SHORT).show();
            }
        });
    }

}

File: 07_05_lab/app/src/main/java/com/example/a07_05_lab/MainActivity.kt
package com.example.a07_05_lab

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val galleryButton = findViewById<Button>(R.id.galleryButton)
        val soundPanelButton = findViewById<Button>(R.id.soundPanelButton)

        galleryButton.setOnClickListener {
            startActivity(Intent(this, FirstActivity::class.java))
        }

        soundPanelButton.setOnClickListener {
            startActivity(Intent(this, SecondActivity::class.java))
        }
    }
}